prosody/Dockerfile

```diff
@@ -2,11 +2,19 @@ FROM jitsi/base
 
 RUN \
 	apt-dpkg-wrap apt-get update && \
-	apt-dpkg-wrap apt-get install -t stretch-backports -y prosody && \
+	apt-dpkg-wrap apt-get install -t stretch-backports -y prosody luarocks gcc libssl1.1 libssl1.0-dev git && \
 	apt-cleanup && \
 	rm -rf /etc/prosody
 
 COPY rootfs/ /
+COPY git-modules/jitsi-meet/resources/prosody-plugins /prosody-plugins
+
+RUN \
+	luarocks list --outdated && \
+	luarocks install luaossl  CRYPTO_LIBDIR=/usr/lib/x86_64-linux-gnu  CRYPTO_INCDIR=/usr/include OPENSSL_LIBDIR=/usr/lib/x86_64-linux-gnu OPENSSL_INCDIR=/usr/include && \
+	luarocks install luajwt CRYPTO_LIBDIR=/usr/lib/x86_64-linux-gnu  CRYPTO_INCDIR=/usr/include OPENSSL_LIBDIR=/usr/lib/x86_64-linux-gnu OPENSSL_INCDIR=/usr/include && \
+	luarocks install luajwtjitsi CRYPTO_LIBDIR=/usr/lib/x86_64-linux-gnu  CRYPTO_INCDIR=/usr/include OPENSSL_LIBDIR=/usr/lib/x86_64-linux-gnu OPENSSL_INCDIR=/usr/include && \
+	luarocks install jwt-jitsi CRYPTO_LIBDIR=/usr/lib/x86_64-linux-gnu  CRYPTO_INCDIR=/usr/include OPENSSL_LIBDIR=/usr/lib/x86_64-linux-gnu OPENSSL_INCDIR=/usr/include
```

Comment:
Do I really need to add `CRYPTO_LIBDIR`, `CRYPTO_INCDIR`, `OPENSSL_LIBDIR` and `OPENSSL_INCDIR `?
I saw it in the issue and just copied it.

Reply:
Please try without them and simplify the code if they are not needed.

Reply:
the OPENSSL and CRYPTO dirs are likely required when building the jwt code, as components of these libraries are used when verifying signatures on JWT tokens.

Reply:
@aaronkvanmeerten sure, but being in standard locations, the env variables may be not required?

Reply:
Updated the PR and removed these variables - I tested it without them and it's working fine.